shared/heroku_app/main.tf

```diff
@@ -11,6 +11,11 @@ variable "name" {
   description = "The application name."
 }
 
+variable "stack" {
```

Comment:
Stack, framework, 🤷‍♂️ Naming is hard.

Reply:
Framework could also make sense! I don't have a strong preference either way.

Reply:
Renamed to `framework` in efb5ff4!

Reply:
🐦